### PR TITLE
updated ru.po

### PR DESCRIPTION
**Pull request by Ser82-png**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -549,15 +545,20 @@ msgstr "Повернуть влево"
...
 msgstr "Повернуть в_лево"
 
 #: pdfarranger/pageutils.py:466 pdfarranger/splitter.py:60
 msgid "Rows"
 msgstr "Ряды"
 
-#: data/menu.ui:232 data/menu.ui:401
+#: pdfarranger/pdfarranger.py:3070
+#, python-format
+msgid "Running on %s"
```
Comment (kbengs) [inline po/ru.po:558]: Left this empty on purpose? The text will be in about dialog. %s is replaced (at runtime) with name of operating system.

**Review by kbengs — COMMENTED** [review]
The PR currently can't be merged because of conflicts.

File: po/ru.po (hunk 1 above)
Comment (Ser82-png) [reply po/ru.po:558]: fixed

**Comment by kbengs** [comment]
Merged as 0f45c293f9e8e904b4d8b53fa986ed21193bc2d3

**Pull request closed without merging**